apps/examples/slsiwifi/Makefile

```diff
@@ -15,40 +15,6 @@
 # language governing permissions and limitations under the License.
 #
 ###########################################################################
-############################################################################
```

Comment:
Old license was wrong. We created the file but someone mistakenly copied the old nuttx header into the file instead of ours. Thats why I removed it. 
All files in these patches have been created by our team.